esignet-core/src/main/java/io/mosip/esignet/core/dto/UserConsent.java

```diff
@@ -0,0 +1,21 @@
+package io.mosip.esignet.core.dto;
+
+import io.mosip.esignet.api.dto.Claims;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+
+@Data
+public class UserConsent {
+    String psuToken;
+    String clientId;
+    Claims Claims;
+    Map<String, Boolean> authorizationScopes;
+    LocalDateTime expiry_dtimes;
```

Comment:
expiry_dtimes, can we change this to expiryDtimes

Reply:
changed it to expirydtimes in line with naming followed with createdtimes